### ISSUE TITLE
Unable to connect to DAXStudio to PowerPivot 

### ISSUE THREAD

**Issue by lucnicod** [DaxStudio/DaxStudio#992]
![image](https://user-images.githubusercontent.com/39966555/201528740-49f37281-2000-427e-aab9-89c20e30a37f.png)

I am trying to connect DAX Studio to a file on OneDrive. I am using Excel beta 2212 (Build 1509.200000). Trying to save the file on the local drive does not help either. Has there been any progress on this issue.
Thanks a lot.
Luc

**Comment by dgosbell** [comment]
Closing this as it is a duplicate of #884 and I'm tracking this issue there. This is an Excel bug that was introduced in the 2208 version and I'm waiting on Excel to release a fix. Once this happens existing copies of DAX Studio will start working again.
